### ISSUE TITLE
botocore requirement needs to be pinned to same version

### ISSUE THREAD

**Issue by nschlemm** [qudos-com/botocore-tornado#2]
Otherwise, one might unknowingly install a current version of botocore which recently introduced breaking api changes described on https://botocore.readthedocs.org/en/latest/client_upgrades.html#client-upgrades

Quick fix for users: explicitly pin botocore to same version as botocore_tornado (currently 0.93.0) in your requirements

Quick fix for maintainer: pin version in https://github.com/qudos-com/botocore-tornado/blob/master/setup.py#L31 stricter

Ideal fix: follow api changes linked above in botocore_tornado

**Comment by sjhewitt** [comment]
good point, thanks for the feedback. I'll make the changes and pin future releases to a fixed botocore version. I'll also update and do a new release compatible with botocore 1.0.0

**Comment by nschlemm** [comment]
ok, cool. thx!

**Comment by sjhewitt** [comment]
I've put a new version (`0.93.1`) on pypi. Unfortunately the update to the new API for `1.0.0` isn't trivial, so I'll have to look at it later when I have a bit more time
